ui/admin/app/routes/scopes/scope/auth-methods/auth-method/managed-groups.js

```diff
@@ -4,17 +4,12 @@
  */
 
 import Route from '@ember/routing/route';
-import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
-import { loading } from 'ember-loading';
-import { confirm } from 'core/decorators/confirm';
-import { notifySuccess, notifyError } from 'core/decorators/notify';
 
 export default class ScopesScopeAuthMethodsAuthMethodManagedGroupsRoute extends Route {
   // =services
 
   @service store;
-  @service intl;
   @service can;
   @service router;
```

Comment:
It looks like router is no longer being used.

Reply:
ahh great catch!